tests/test_font.py

```diff
@@ -58,6 +89,15 @@ def test_FontArguments_getCollectionIndex(fontarguments):
     assert isinstance(fontarguments.getCollectionIndex(), int)
 
 
+def test_FontArguments_getPalette(fontarguments):
+    palette = fontarguments.getPalette()
+    assert isinstance(palette, skia.FontArguments.Palette)
```

Comment:
Ditto about fixture and multiple tests.

Reply:
There is only one tests here. The asserts are all checking the same palette to make sure it is a valid one.

Reply:
> If the asserts are not logically independent I.e. you just want an exact match of a composite structure, it might be clearer to do assert x and y and z a single mulit-line assert, any mis-match is considered a failure. If there are multiple independent/stages ways of failure, you do a fixture then multiple tests

I don’t see how `assert isinstance(palette, skia.FontArguments.Palette) and isinstance(palette.overrides, skia.FontArguments.Palette.Overrides) and (palette.index == 0) and (len(palette.overrides) == 0) and (palette.overrideCount == 0)` is an improvement over what I have here. I don’t know what to put in a fixer, if I were to use one, since there is no multiple inputs here. It is getting an empty palette and asserting that it is indeed a palette and that it is empty.

Reply:
I removed the extra asserts, they are not really needed.